xarray/backends/rasterio_.py

```diff
@@ -144,8 +144,13 @@ def open_rasterio(filename, chunks=None, cache=None, lock=None):
     attrs = {}
     if hasattr(riods, 'crs'):
         # CRS is a dict-like object specific to rasterio
-        # We convert it back to a PROJ4 string using rasterio itself
-        attrs['crs'] = riods.crs.to_string()
+        if riods.crs is None:
+            # If crs is None we put empy string to attrs list
+            attrs['crs'] = ''
```

Comment:
Is this more useful than not simply dropping the attribute?

Reply:
I'm not really sure, since I don't know what it does in the first place. I'll look into it on Monday when I'm back at work.

Reply:
My vote would be to keep this simple. 
```Python
if hasattr(riods, 'crs') and riods.crs:
    attrs['crs'] = riods.crs
```

Reply:
After looking into documentation of CRS-parateter, I have to agree that it should not be empty string. I will commit changes in 10 minutes.